### PR TITLE
Http request for pylib

### PR DESCRIPTION
**Pull request by aslepakurov**
- Verbose merged script
- Http request parse
- Tests

### CODE REVIEW

**Review by rozenbom — COMMENTED** [review]
Typo https://github.com/verygoodsecurity/starlarky/blob/1303b0195f5385f1eddf958e79b928a0170f4bcf/pylarky/eval/evaluator.py#L4

File: pylarky/eval/http_evaluator.py [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
 from pylarky.eval.evaluator import Evaluator
-from pylarky.request import HttpRequest
+from pylarky.model.request import HttpRequest
 
 
 class HttpEvaluator(Evaluator):
 
-    def __init__(self, script, http_request: HttpRequest):
-        super(HttpEvaluator, self).__init__(script, http_request.to_starlark())
+    def __init__(self, script):
```
Comment (rozenbom) [inline pylarky/eval/http_evaluator.py:10]: BTW why do you need __init__ here at all?

File: setup.py [hunk 2]
```diff
@@ -1,10 +1,18 @@
+import os
 from setuptools import find_packages, setup
 
+
+version = 'nover'
+try:
+    version = os.environ['CIRCLE_TAG']
```
Comment (rozenbom) [inline setup.py:7]: Not sure that hardcoding CI-specific staff is the right way

**Review by mahmoudimus — APPROVED** [review]
This looks good to me

File: pylarky/eval/http_evaluator.py [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
+        super(HttpEvaluator, self).__init__(script)
+
+    def evaluate(self, http_request: HttpRequest) -> HttpRequest:
```
[the remark this answers is not in the document]
Comment (aslepakurov) [reply pylarky/eval/http_evaluator.py:13]: I will change naming, as the structure of request should be the same - payload and headers

File: pylarky/eval/http_evaluator.py (hunk 1 above)
Comment (aslepakurov) [reply pylarky/eval/http_evaluator.py:10]: For possible future usage of using same script for multiple payloads

**Pull request merged**